tests/App/Action/IntegrationTestCase.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\ClassificationBundle\Tests\App\Action;
+
+use Doctrine\ORM\EntityManagerInterface;
+use Sonata\ClassificationBundle\Tests\App\AppKernel;
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+use Symfony\Component\Console\Input\ArrayInput;
+
+abstract class IntegrationTestCase extends WebTestCase
+{
+    protected function setUp(): void
+    {
+        $kernel = self::createKernel();
+        $kernel->boot();
+
+        $application = new Application($kernel);
+        $application->setAutoExit(false);
+
+        $application->run(new ArrayInput([
```

Comment:
Would be nice to copy what has been done on doctrine orm admin bundle and mongodb admin bundle. To not introduce another way of doing this.